tests/E2E/ErrorTest.php

```diff
@@ -40,20 +40,23 @@ public function testAccessTokenInvalid()
      */
     public function testInvalidQuery()
     {
-        $client = new Client('b4c0n73n7fu1', 'cfexampleapi');
+        $client = $this->getClient('cfexampleapi');
+
         $query = (new Query())
             ->where('name', 0);
 
         $client->getEntries($query);
     }
 
     /**
+     * @requires Contentful
```

Comment:
Standard `@requires` annotations have a format `@requires SUBJECT ATTRIBUTE`, such as `@requires PHP 7.0` or `@requires extension ext-soap`. I suggest we use a similar structure for consistency, maybe `@requires API no-proxy`